main.go

```diff
@@ -357,3 +361,36 @@ func setupKcpWatcherReconciler(mgr ctrl.Manager, options controller.Options, fla
 		os.Exit(1)
 	}
 }
+
+func dropV1alpha1FromStoredVersions(mgr manager.Manager) {
+	cfg := mgr.GetConfig()
+	kcpClient, err := apiextension.NewForConfig(cfg)
+	if err != nil {
+		setupLog.V(log.DebugLevel).Error(err, "unable to initialize client to remove v1alpha1")
+	}
+	ctx := context.TODO()
+	var crdList *v1extensions.CustomResourceDefinitionList
+	if crdList, err = kcpClient.ApiextensionsV1().CustomResourceDefinitions().List(ctx, v1.ListOptions{}); err != nil {
+		setupLog.V(log.DebugLevel).Error(err, "unable to list CRDs")
+	}
+
+	for _, crdItem := range crdList.Items {
+		if crdItem.Spec.Group != "operator.kyma-project.io" {
+			continue
+		}
+		oldStoredVersions := crdItem.Status.StoredVersions
+		newStoredVersions := make([]string, 0, len(oldStoredVersions))
+		for _, stored := range oldStoredVersions {
+			if stored != "v1beta2" {
```

Comment:
you should remove v1alpha1, but not v1beta2

Reply:
@ruanxin Yes I had this pushed by mistake, thanks!